avatar/avatar.css

```diff
@@ -0,0 +1,189 @@
+@import url("https://fonts.googleapis.com/css2?family=Lato&family=Merriweather:wght@300&family=Montserrat&family=Nanum+Gothic&family=Noto+Serif&family=Quicksand&family=Raleway:wght@200&family=Roboto+Slab:wght@300&family=Rubik:wght@300&family=Source+Sans+Pro&family=Ubuntu:wght@300&family=Work+Sans:wght@200&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@300&display=swap");
+
+:root {
+   font-family: "Noto Serif", serif;
+
+   --primary-color: #002852;
+   --secondary-color: white;
+   --black: #001224;
+   --footer-black: #1e2022;
+   --primary-padding: 2rem;
+}
+
+* {
+   margin: 0;
+   padding: 0;
+   box-sizing: border-box;
+   /* overflow-x: hidden; */
```

Comment:
Remove unwanted code comments from the whole code base, it makes code look even better and avoids accidental error or bug generation.